tests/playwright/src/utility/fixtures.ts

```diff
@@ -0,0 +1,43 @@
+import type { Page } from '@playwright/test';
+import { test as base } from '@playwright/test';
+
+import { WelcomePage } from '../model/pages/welcome-page';
+import { NavigationBar } from '../model/workbench/navigation';
+import { PodmanDesktopRunner } from '../runner/podman-desktop-runner';
+
+export type TestFixtures = {
+  pdRunner: PodmanDesktopRunner;
+  navBar: NavigationBar;
+  welcomePage: WelcomePage;
+  page: Page;
+};
+
+export type PDRunnerOptions = {
```

Comment:
```suggestion
export type PodmanDesktopRunnerOptions = {
```

Reply:
(or just get rid of PodmanDesktop prefix as we're anyway inside Podman Desktop)

Reply:
but usually when you import stuff there is no prefix in the imports

like

```
import {Foo} from 'podman-desktop-library'
```

not
```
import {PodmanDesktopFoo} from 'podman-desktop-library'
```

it's like `test` you import `test` from `playwright` or from `jest` or `vitest` the prefix is in fact the namespace that we import

and if you want to get rid of some ambiguities you change the name when doing the import but it's a consumer choice

Reply:
Renamed to RunnerOptions. :)

Reply:
yes I think PodmanDesktopRunner should be exported also under Runner (so for a while both can be imported) and here RunnerOptions

```ts
import { Runner, RunnerOptions} from '@podman-desktop/tests-playwright';
```

Reply:
Going to create a different issue to have the runner renamed, don't want to have it here, PR is long enough as it is.

Reply:
Created https://github.com/containers/podman-desktop/issues/8749 to track the runner renaming.

Reply:
sure, thanks

Reply:
I think the PR is longer also due to all the `navBar` usage instead of `navigationBar`

Reply:
yeah it's true, we didn't actually have a standard for it, half of the test files were one way and half the other. :/

Reply:
if you prefer you can just issue a PR doing a change to have something uniform between all `navBar` or `navigationBar` as sometimes `navBar` is currently used or `navigationBar`